lib/mongodb.js

```diff
@@ -0,0 +1,46 @@
+import { MongoClient } from 'mongodb'
+
+let uri = process.env.MONGODB_URI
+let dbName = process.env.MONGODB_DB
+let cachedClient = null
+let cachedDb = null
+
+if (!uri) {
+  throw new Error(
+    'Please define the MONGODB_URI environment variable inside .env.local'
+  )
+}
+
+if (!dbName) {
+  throw new Error(
+    'Please define the MONGODB_DB environment variable inside .env.local'
+  )
+}
+
+async function connectToDatabase() {
+  if (cachedClient && cachedDb) {
+    return { client: cachedClient, db: cachedDb }
+  }
+
+  const client = await MongoClient.connect(uri, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+
+  const db = await client.db(dbName)
+
+  cachedClient = client
+  cachedDb = db
+
+  return { client, db }
+}
+
+export default async function mongo(collection, find = {}) {
```

Comment:
Did you add this function?

Reply:
yes
Makes stuff easier :)

Reply:
okay. I'll look at it more closely and think about naming. Right now it is not descriptive enough and there is no commenting telling what it does either.